net/wol/src/opnsense/www/js/widgets/Metadata/WakeOnLan.xml

```diff
@@ -0,0 +1,12 @@
+<metadata>
+    <wol>
+        <filename>WakeOnLan.js</filename>
+        <endpoints>
+           <endpoint>/api/wol/wol/*</endpoint>
+           <endpoint>/diagnostics/interface/*Arp</endpoint>
```

Comment:
It's better to list everything here vs. using `*Arp` for future robustness.

Reply:
Fixed in 1d700fb39e8a3bfc4fe7632654402c64f66fa2d5